### ISSUE TITLE
UI Doesn't show Boss Money

### ISSUE THREAD

**Issue by HumanTree92** [HumanTree92/esx_customui#2]
This is currently being worked on. Comment Below for Boss Money info only!!!

**Comment by chubbynugly** [comment]
it's funny your sitting here editing code that was leaked, gg

**Comment by HumanTree92** [comment]
@chubbynugly Oh yes editing code that was leaked but not working properly. I have fixed it & added better stuff to it.

**Comment by chubbynugly** [comment]
It wasn't supposed to be released!

**Comment by HumanTree92** [comment]
And your point?

**Comment by chubbynugly** [comment]
Have some respect is my point

**Comment by HumanTree92** [comment]
Respect for code that was shared on FiveM by 3 different people. I have found over 5 GitHub repositories that have this script. So your telling me that me doing the same is wrong. Me taking code that wasn't working & fixing it & adding things to better help the community of FiveM is wrong. Gotcha GG kid

**Comment by HumanTree92** [comment]
Closing b/c i don't need disrespect from a 12yr old

**Comment by HumanTree92** [comment]
By the way leaked i wouldn't say. Someone from ADRP released it to the Public to use & do whatever they want with. As you can see at the bottom of [this](https://github.com/Chubbz123/fivem-ui) meaning whoever this person is had the code & gave it out. Not leaked. If it was leaked then the people of ADRP would have taken it down.

**Comment by chubbynugly** [comment]
I am the owner of ADRP they used cache decrypter to steal our files.

**Comment by HumanTree92** [comment]
oh... Which ADRP are you?

**Comment by chubbynugly** [comment]
American Dream Roleplay, where it came from?

**Comment by HumanTree92** [comment]
Ah. Yes i used to play it there a long time ago. So https://github.com/Chubbz123/fivem-ui ain't you.

**Comment by chubbynugly** [comment]
No, it is not. Wish it was so I could remove it

**Comment by HumanTree92** [comment]
So you created the code correct?

**Comment by chubbynugly** [comment]
http://prntscr.com/kqap9w

**Comment by HumanTree92** [comment]
Also found this one [here](https://github.com/SGTGunner/fivem-ui) Alright well is there some way we can come to an agremeant?

**Comment by chubbynugly** [comment]
thats just a fork, with it broken I would like it to stay as the chubbz123 repo

**Comment by HumanTree92** [comment]
So there is no way that we can come to agrement for me to continue to work on it & keep it released for others to use?

**Comment by chubbynugly** [comment]
It wasn't supposed to be released so if you would like to keep it up you'd be doing it against the developers wishes.

**Comment by HumanTree92** [comment]
So your not the Developer of it?

**Comment by chubbynugly** [comment]
I am, i'm saying if you keep it up you will be going against my wishes

**Comment by HumanTree92** [comment]
I would much rather keep it up. Before i even updated it & before the leaks i have seen this UI on other servers. 

Plus there are lots of others that like this UI & i would have lost 3 days of editing this script for others for no reason.

**Comment by HumanTree92** [comment]
I have made stuff myself yes. The one i made for my server didn't look anything like that but looked like crap. I was browsing through a support ticket on esx github not sure which one & i had seen someone else using this UI.  I had asked him where he got it from & he said he was the creator of it but sent me to a topic on FiveM that he released it on. So you saying that you made it & him saying he made it not sure who to believe to be honest. But it was released on GitHub. I'm just saying that yes if i have valid proof that you or him own it i will take it down. I will try & find the guy i talked to & see if he has proof. All i did was take somewhat broken code (missing 1 small line of code) & fixed it then started adding things to it.
